e2e/rbd.go

```diff
@@ -1089,6 +1089,45 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a PVC and bind it to an app with ext4 as the FS and 1024 inodes ", func() {
```

Comment:
@nixpanic can we have  a test case for XFS FS as well ?

Reply:
I do not think there is much value in that. XFS is used in other tests already, so this PR does not break it.